src/attr/validators.py

```diff
@@ -592,3 +594,117 @@ def min_len(length):
     .. versionadded:: 22.1.0
     """
     return _MinLengthValidator(length)
+
+
+@attrs(repr=False, slots=True, hash=True)
+class _SubclassOfValidator:
+    type = attrib()
+
+    def __call__(self, inst, attr, value):
+        """
+        We use a callable class to be able to change the ``__repr__``.
+        """
+        if not issubclass(value, self.type):
+            raise TypeError(
+                "'{name}' must be a subclass of {type!r} (got {value!r} "
+                "that is a {actual!r}).".format(
+                    name=attr.name,
+                    type=self.type,
+                    actual=value.__class__,
+                    value=value,
+                ),
+                attr,
+                self.type,
+                value,
+            )
+
+    def __repr__(self):
+        return "<subclass_of validator for type {type!r}>".format(
```

Comment:
ci is failing because this line is missing coverage. 🤓

Reply:
Can I get privileges to run CI? 😉

Reply:
crypto bros ruined that for everybody :( you’ll be in after this is merged

Reply:
(takes one merged PR)